lib/ember_cli/build_monitor.rb

```diff
@@ -32,7 +32,7 @@ def wait!
     attr_reader :name, :paths
 
     def complete?
-      !paths.lockfile.exist?
+      !paths.lockfile.exist? && paths.dist.join("index.html").exist?
```

Comment:
Yes, it does fix my issue.

I couldn't find where the lockfile gets removed... I looked but no luck. More correct behavior would be to wait until index.html exists in dist before the lockfile gets removed. 

Overall, expected behavior is that index.html should exist before declaring build done though, right? Open to suggestions on where to ensure that logic.